ada_feeding/ada_feeding/trees/acquire_food_tree.py

```diff
@@ -9,35 +9,39 @@
 from typing import List, Optional
 
 # Third-party imports
+from geometry_msgs.msg import Twist, TwistStamped
 from overrides import override
 import py_trees
 from py_trees.blackboard import Blackboard
 import py_trees_ros
 from rcl_interfaces.srv import SetParameters
 from rclpy.node import Node
+from rclpy.qos import QoSProfile
+from std_msgs.msg import Header
 
 # Local imports
 from ada_feeding import ActionServerBT
+from ada_feeding.behaviors import ToggleCollisionObject, UpdateTimestamp
 from ada_feeding.behaviors.acquisition import (
     ComputeFoodFrame,
-    ComputeApproachConstraints,
-    ComputeExtractConstraints,
+    ComputeActionConstraints,
+    ComputeActionTwist,
 )
 from ada_feeding.behaviors.moveit2 import (
     MoveIt2JointConstraint,
-    MoveIt2OrientationConstraint,
     MoveIt2PoseConstraint,
-    MoveIt2PositionConstraint,
     MoveIt2Plan,
     MoveIt2Execute,
 )
+from ada_feeding.decorators import TimeoutFromBlackboard
 from ada_feeding.helpers import BlackboardKey
 from ada_feeding.idioms import (
     pre_moveto_config,
     scoped_behavior,
     retry_call_ros_service,
 )
 from ada_feeding.idioms.pre_moveto_config import set_parameter_response_all_success
+from ada_feeding.trees import StartServoTree, StopServoTree
 from ada_feeding.visitors import MoveToVisitor
 from ada_feeding_msgs.action import AcquireFood
 from ada_feeding_msgs.srv import AcquisitionSelect
```

Comment:
Heads-up that some versions of `pylint` view imports from `ada_feeding_msgs` as "Third-party" and not as "Local." To address this, I'd recommend putting inputs from `ada_feeding_msgs` at the top of the "Local" block, to accommodate both versions of `pylint`.